Yoast/wordpress-seo
# Redesign the Help Center support tab.

## Summary

This PR can be summarized in the following changelog entry:

* Redesigned the Help Center support tab for Premium.

## Test instructions

This should be tested on Premium. On the free version you can see the new design temporarily changing `$this->premium_support` to `1` but I guess the best way would be making a diff and apply it on Premium. DOn't forget to build the CSS and JS.

Notes:
- not sure how to make the text "100% secure" bold, suggestione welcome
- in the CSS the part with `nth-child` selectors is optional just to match in a pixel-perfect way the design provided. I'd rather remove that, as we can't replicate the paragraph widths as in the design, especially when they will be translated.

Fixes https://github.com/Yoast/yoast-components/issues/341
